kaska/parmap.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Lifted from https://gist.github.com/Jwink3101/494e741f07d33edea47d369bcfc4a54a"""
-from __future__ import division, print_function, unicode_literals, absolute_import
+"""Lifted from:
+ https://gist.github.com/Jwink3101/494e741f07d33edea47d369bcfc4a54a"""
+from __future__ \
+    import division, print_function, unicode_literals, absolute_import
```

Comment:
We are now running python 3 everywhere (or we should be!). These imports from future can be removed.

Reply:
Fixed